posts/2020-12-16-rust-survey-2020.md

```diff
@@ -64,7 +64,7 @@ While Rust usage seems to be growing at a healthy pace, the results of the surve
 
 ### C++ Interop
 
-Interestingly, C++ interop was the most requested language for better interop with Rust with C and Python in second and third place. Improved C++ interop was especially often mentioned as a way to improve Rust usage specifically at work. In fact, for users who work on large codebases (100,000 lines of code or larger) C++ interop and unsurprisingly compile times were the most cited ways to improve their Rust experience.    
+Interestingly, C++ was the most requested language for better interop with Rust, with C and Python in second and third place. Improved C++ interop was especially often mentioned as a way to improve Rust usage specifically at work. In fact, for users who work on large codebases (100,000 lines of code or larger), C++ interop and – unsurprisingly – compile times were the most cited ways to improve their Rust experience.    
```

Comment:
em dashes, not en dashes, please

Reply:
Do you also want spaces around the em dashes, or no?

Reply:
yes